bower.json

```diff
@@ -23,7 +23,10 @@
     "paper-progress": "PolymerElements/paper-progress#^1.0.0",
     "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "iron-behaviors": "PolymerElements/iron-behaviors#^1.0.0"
+    "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
+    "iron-behaviors": "PolymerElements/iron-behaviors#^1.0.0",
+    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.1",
```

Comment:
This technically shouldn't have to be anything other than `x.y.0`, where `x` is major version and `y` is minor version. Just observing, no action required.